deps/libgit2.mk

```diff
@@ -40,6 +40,10 @@ ifeq ($(OS),Linux)
 LIBGIT2_OPTS += -DUSE_OPENSSL=OFF -DUSE_MBEDTLS=ON -DCMAKE_INSTALL_RPATH="\$$ORIGIN"
 endif
 
+ifeq ($(OS),FreeBSD)
+LIBGIT2_OPTS += -DCMAKE_INSTALL_RPATH="\$$ORIGIN"
```

Comment:
would extending this section https://github.com/JuliaLang/julia/blob/d7625dd7dccab548025fa8b5b9d4c7941336fb24/deps/libgit2.mk#L47-L63 to also apply to freebsd fix the startup issue with mbedtls?